static/i18n/en/app.json

```diff
@@ -1220,6 +1220,9 @@
     "FeeRequired": {
       "title": "Fees are required"
     },
+    "GasLessThanEstimate": {
+      "title": "Provided gas limit is less than the estimated"
```

Comment:
@dasilvarosa can you think of a better wording?
it's a warning that displays under the gas limit field when user lower the gas limit under the safe estimated one. (setting that low value is likely to fail the transaction)

if we can not say "gas limit" in the wording it's better so we can later reuwe this concept. if you think we need to mention it, still fine.

Reply:
This may be too low, please increase

Reply:
thanks